src/test/java/co/rsk/federate/signing/hsm/client/HSMSignatureTest.java

```diff
@@ -18,6 +18,7 @@
 
 package co.rsk.federate.signing.hsm.client;
 
+import co.rsk.federate.signing.utils.TestUtils;
 import org.ethereum.crypto.ECKey;
 import org.ethereum.crypto.HashUtil;
```

Comment:
Is this still being used?

Reply:
No it isn't. I have disabled import optimizations in the IDE. Solved!